### PR TITLE
fix language selection

### PR DESCRIPTION
**Pull request by laserchicken**
request: https://github.com/lonelyplanet/rizzo/issues/1640

### CODE REVIEW

File: app/assets/javascripts/lib/core/language_nav.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+define([ "jquery" ], function($) {
+
+  "use strict";
+
+  function LanguageNav() {
+
+    $("#languageSelect").on("change", function(event) {
```
Comment (WunderBart) [inline app/assets/javascripts/lib/core/language_nav.js]: Can we use `.js-language-select` here instead?

File: app/assets/javascripts/lib/core/language_nav.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      var url = "http://" + $(this).val();
```
Comment (WunderBart) [inline app/assets/javascripts/lib/core/language_nav.js]: Sure we want to force `http`?

**Comment by laserchicken** [comment]
updated @WunderBart

File: app/assets/javascripts/lib/core/language_nav.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  LanguageNav.prototype.listen = function() {
+    this.$el.on("change", this._handleSelect);
+  };
+
+  LanguageNav.prototype._handleSelect = function(event) {
+    event.preventDefault();
+    var url = "http://" + $(this).val();
+    window.location = url;
```
Comment (WunderBart) [inline app/assets/javascripts/lib/core/language_nav.js]: I don't understand this one - do we redirect via `window.location` or [redirector form](https://github.com/lonelyplanet/rizzo/pull/1646/files#diff-0f67acf0410f857c2da18ffdc1b850a3R33)?

Comment (laserchicken) [reply app/assets/javascripts/lib/core/language_nav.js]: Sorry, I should remove all form related params - updated.
We are not using redirector form anymore, just simple dropdown select, and redirect with `window.location`, exactly the same as it's implemented in [rizzo-next](https://github.com/lonelyplanet/rizzo-next/blob/master/src/components/footer/footer_component.js#L16)

Comment (laserchicken) [reply app/assets/javascripts/lib/core/language_nav.js]: Redirector endpoint (after we switched to https [here](https://github.com/lonelyplanet/rizzo/commit/a51be21a0ad992059248280b54a1e76c10f40e65#diff-0f67acf0410f857c2da18ffdc1b850a3R43)) started forcing https for all redirects and some of those external pages do not support it. We need to make it work and the only option is  to change it back to http for now (and we can not use redirector endpoint anymore).

Comment (laserchicken) [reply app/assets/javascripts/lib/core/language_nav.js]: Does it make sense?

**Pull request closed without merging**